app/src/main/java/com/libs/facerecognition/models/FaceNetModel.java

```diff
@@ -0,0 +1,33 @@
+package com.libs.facerecognition.models;
+

```

Comment:
I strongly disagree with this solution. We should have one generic class, which will have model dictionary. GetInstance(nameOfModelFile) should return instance of this model if exist or create new one and add it to dictionary. Iamgine adding 10 models now...

Reply:
+1